src/pages/OPP_SendAcknowledgmentBTN.page

```diff
@@ -2,8 +2,6 @@
     <apex:sectionHeader title="{!$Label.sendAcknowledgmentPageTitle}" />
     <apex:pageMessages />
     <p/>
-    <apex:outputLink value="{!retURL}">{!$Label.sendAcknowledgmentReturnLinkLabel}</apex:outputLink>
```

Comment:
@njjc because it shows the return link 2x when there are no errors, when there are errors it puts the link at the top and the bottom and we decided to just remove the top link.